client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -68,15 +68,15 @@ export const UserProfilePage = () => {
           <Heading as="h2" size={'lg'}>
             Welcome, {getNameText(userInfo.name)}
           </Heading>
-          {userInfo.admined_chapters.length > 0 && (
+          {userInfo.user_chapters.length > 0 && (
             <>
               <Heading as="h2" marginBlock={'.5em'} size="md">
-                You are an administrator for these Chapters:
+                You are an member of these Chapters:
               </Heading>
               <Flex marginTop={'1em'} flexDirection={'column'} gap={4}>
-                {userInfo.admined_chapters.map(({ name, id }) => (
-                  <Link key={id} href={`/chapters/${id}`}>
-                    {name}
+                {userInfo.user_chapters.map(({ chapter }) => (
+                  <Link key={chapter.id} href={`/chapters/${chapter.id}`}>
+                    {chapter.name}
```

Comment:
Why was this changed?  It's not connected to the download.

Reply:
Because, the data is the one change as it isn't related to chapters that admin is administrating, but chapters that member has joined

Reply:
Okay, so, we have both `admined_chapters` and `user_chapters` now, so, for this PR it would make most sense to leave this unchanged (since the scope should just be adding the download button).

A separate PR that shows both on the profile page would be good.